src/LiquidityManager.sol

```diff
@@ -0,0 +1,189 @@
+// SPDX-License-Identifier: GPL-3.0-or-later
+pragma solidity ^0.8.13;
+
+import { RMM, IPYieldToken } from "./RMM.sol";
+import { ERC20 } from "@openzeppelin/contracts/token/ERC20/ERC20.sol";
+import {IStandardizedYield} from "pendle/interfaces/IStandardizedYield.sol";
+import {PYIndexLib, PYIndex} from "pendle/core/StandardizedYield/PYIndex.sol";
+import {FixedPointMathLib} from "solmate/utils/FixedPointMathLib.sol";
+
+import "./lib/RmmErrors.sol";
+
+interface IRMM {
```

Comment:
Since we already import RMM, we can cast it to RMM

Reply:
That was causing weird stack issues with via-ir, no clue why